### PR TITLE
Add babelforce to ADOPTERS.md

### PR DESCRIPTION
**Pull request by nik843**
### Description of your changes

Add babelforce as Adopter of Crossplane

I have: 

- [x] Read and followed Crossplane's [contribution process].
- [ ] ~Run `make reviewable` to ensure this PR is ready for review.~
- [ ] ~Added or updated unit tests.~
- [ ] ~Added or updated e2e tests.~
- [ ] ~Linked a PR or a [docs tracking issue] to [document this change].~
- [ ] ~Added `backport release-x.y` labels to auto-backport this PR.~

### CODE REVIEW

File: ADOPTERS.md [hunk 1]
```diff
@@ -73,3 +73,4 @@ This list is sorted in the order that organizations were added to it.
 | [Printbox](https://getprintbox.com/) | [@printbox-team](https://github.com/printbox-team) | Printbox is leveraging Crossplane in production environments, to streamline the deployment of a wide range of services. Our usage of Crossplane extends from basic tasks like creating S3 buckets on AWS to more complex operations such as implementing CDN solutions on Google Cloud. |
 | [DeepSea](https://deepsea.ai/) | [bmutziu](https://github.com/bmutziu) | Migrated to Crossplane from Terraform in production environments. |
 | [Imagine Learning](https://www.imaginelearning.com/) | [@blakeromano](https://github.com/blakeromano) [blake.romano@imaginelearning.com](mailto:blake.romano@imaginelearning.com) | Control Plane for Infrastructure in Internal Developer Platform |
+| [babelforce](https://www.babelforce.com/) | [@nik843](https://github.com/nik843) |Orchestrating relational database servers by creating databases, users and permissions for them within all environments. |
```
Comment (phisco) [inline ADOPTERS.md:76]:
```suggestion
| [babelforce](https://www.babelforce.com/) | [@nik843](https://github.com/nik843) | Orchestrating relational database servers by creating databases, users and permissions for them within all environments. |
```

**Review by phisco — APPROVED** [review]
Thanks @nik843, just a minor nit!

File: ADOPTERS.md (hunk 1 above)
Comment (nik843) [reply ADOPTERS.md]: should I rebase my branch?

**Comment by nik843** [comment]
should I rebase my branch?

**Comment by negz** [comment]
@nik843 Thank you! I think I can squash merge your PR, so no need to rebase.

**Review by negz — APPROVED** [review]

**Pull request merged**